src/components/Dashboard/StaffNotation.vue

```diff
@@ -240,8 +236,7 @@ export default class StaffNotation extends Vue {
   /** Prop for the scrollbar offset to be added. */
   @Prop() readonly addScrollbarOffset!: string
 
-  /** Prop for disabling the menu items. */
-  @Prop({ default: false }) readonly disabled!: boolean
+  @Action setFetchingDataSpinner!: ActionBindingIF
```

Comment:
Can we use the Mutation instead?

Reply:
Yes! You'll see this in the next commit.